pkgs/development/tools/pyoxidizer/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, pkgconfig
+, python38
+, openssl
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "PyOxidizer";
+  version = "0.10.3";
+
+  src = fetchFromGitHub {
+    owner = "indygreg";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "07hd00nyg1mbjlg8241ci5w6clqchcrlmld2vy4nca26j5b4k46k";
+  };
+
+  cargoSha256 = "14k81n56w29fdwm1bwbsmb3p7nawjbvfrqg2p74a0bz33hq5iby3";
+  depsBuildBuild = [ openssl pkgconfig python38 ];
+  doCheck = false;
```

Comment:
The whole `meta` section is missing i.e.
```
  meta = with stdenv.lib; {
    description = "A utility for producing binaries that embed Python";
    homepage = "https://github.com/indygreg/PyOxidizer";
    license = licenses.mpl20;
    maintainers = with maintainers; [ ??? ];
    platforms = ???
```
You'll also need to add yourself as a maintainer to the `maintainers/maintainer-list.nix`, [manual](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes) for further information on the meta attribute.